index.bs

```diff
@@ -1350,6 +1352,41 @@ To <dfn for="XRView">obtain the projection matrix</dfn> for a given {{XRView}} |
 </div>
 
 
+Primary and Secondary Views {#primary-and-secondary-views}
+----------
+
+A [=view=] is a <dfn>primary view</dfn> if rendering to it is necessary for the immersive experience.
+
+A [=view=] is a <dfn>secondary view</dfn> if it is possible for content to choose to not render to it and still produce a working immersive experience. If content chooses to not render to these views, the user-agent MAY reconstruct them via reprojection.
+
+<div class="example">
+Examples of [=primary views=] include the main mono view of a handheld AR session, the main two stereo views of headworn AR/VR sessions, or all of the wall views of a CAVE session.
+
+Examples of [=secondary views=] include the [=first-person observer view=] used for video capture, or "quad views" where there are two views per eye with differing resolutions or fields of view.
+
+</div>
+
+<div class=note>
+While content should be written to assume that there may be any number of views, we expect a significant amount of content to make incorrect assumptions about the {{XRViewerPose/views}} array and thus break when presented with more than two views.
```

Comment:
Recognizing that https://github.com/immersive-web/webxr-ar-module/pull/57/files is still in progress, is there a compelling reason to identify which views are primary/secondary in cases other than the first person observer scenario? My gut says yes, but I can't identify a scenario where you'd need to know other than the first person observer.

Reply:
Some other cases I've heard discussed:
* Third-person observer (e.g. similar to [Spectator View](https://docs.microsoft.com/en-us/windows/mixed-reality/spectator-view), although that particular implementation uses app-side state sync to a different device rather than this secondary views mechanism on one device)
* 360-degree screenshot/video (triggered by user in system UI)

Reply:
@toji I don't see a good reason to identify bionic (quad) views. I think third-person observer would need to be _independently_ identified.